README.md

```diff
@@ -145,91 +220,54 @@ kurtosis enclave inspect YOUR_ENCLAVE_ID_HERE
 Kurtosis will output everything it knows about your enclave, similar but not identical to the output below:
 
 ```
-Enclave ID:                KT2021-12-08T13.22.31.404
-Enclave Status:            EnclaveContainersStatus_RUNNING
-API Container Status:      EnclaveAPIContainerStatus_RUNNING
-API Container Host Port:   127.0.0.1:50828
-
-======================================== Interactive REPLs ========================================
-GUID
-1638991359
+========================================= Kurtosis Modules =========================================
+GUID                                             LocalPortBindings
+ethereum-kurtosis-module.1649105153-1649105156   1111/tcp -> 127.0.0.1:55897
 
 ========================================== User Services ==========================================
-GUID                         LocalPortBindings
-bootnode_1638991384          8545/tcp -> 127.0.0.1:50845
-                             8546/tcp -> 127.0.0.1:50846
-                             30303/tcp -> 127.0.0.1:50847
-                             30303/udp -> 127.0.0.1:60255
-ethereum-node-1_1638991388   8545/tcp -> 127.0.0.1:50853
-                             8546/tcp -> 127.0.0.1:50851
-                             30303/tcp -> 127.0.0.1:50852
-                             30303/udp -> 127.0.0.1:57127
-ethereum-node-2_1638991392   8545/tcp -> 127.0.0.1:50948
-                             8546/tcp -> 127.0.0.1:50949
-                             30303/tcp -> 127.0.0.1:50947
-                             30303/udp -> 127.0.0.1:49317
-
-========================================= Kurtosis Modules =========================================
-GUID                    LocalPortBindings
-eth-module_1638991377   1111/tcp -> 127.0.0.1:50841
+GUID                         ID                LocalPortBindings
+bootnode-1649105160          bootnode          30303/tcp -> 127.0.0.1:55902
+                                               30303/udp -> 127.0.0.1:49555
+                                               8545/tcp -> 127.0.0.1:55903
+                                               8546/tcp -> 127.0.0.1:55901
+ethereum-node-1-1649105164   ethereum-node-1   8545/tcp -> 127.0.0.1:55907
+                                               8546/tcp -> 127.0.0.1:55908
+                                               30303/tcp -> 127.0.0.1:55909
+                                               30303/udp -> 127.0.0.1:56275
+ethereum-node-2-1649105167   ethereum-node-2   30303/udp -> 127.0.0.1:51099
+                                               8545/tcp -> 127.0.0.1:55964
+                                               8546/tcp -> 127.0.0.1:55965
+                                               30303/tcp -> 127.0.0.1:55963
 ```
+Now let's connect to the http address exposed in the bootnode to retrieve Ethereum network information.
```

Comment:
"Now let's connect to the Ethereum bootnode to verify the network is producing blocks.

Find the Ethereum node with ID `bootnode` in the enclave contents, find its RPC port declared on `8545/tcp`, and copy the the public IP and port that it's bound to on your machine (e.g. `127.0.0.1:55903`).

Then, slot it into `RPC_URL_HERE` in the below command:
"